source/gpus_in_openstack.rst

```diff
@@ -124,13 +124,7 @@ Create a new playbook or update an existing on to apply the roles:
               kayobe_needs_reboot: true
 
       - name: Reboot when required
-        hosts: overcloud
-        vars:
-          # This playbook will execute after nodes are deployed
-          # and before overcloud host configure - we can't assume
-          # users and venvs exist.
-          ansible_user: "cloud-user"
-          ansible_python_interpreter: "/usr/bin/python3"
+        hosts: iommu:vgpu
```

Comment:
These groups also need to be added to the inventory.

Reply:
done